components/Layout.js

```diff
@@ -0,0 +1,44 @@
+import styled from "styled-components";
+
+import { TitleBar } from "./TitleBar";
+
+export function Layout({ children }) {
+  return (
+    <Wrapper>
+      <TitleBar></TitleBar>
+      <StyledMain>{children}</StyledMain>
+      <Footer>
+        <Copyright>&copy; OM</Copyright>
+      </Footer>
+    </Wrapper>
+  );
+}
+
+const Wrapper = styled.div`
+  display: grid;
+  grid-template-rows: 80px auto 36px;
+  height: 100vh;
+`;
+
+const StyledMain = styled.main`
+  overflow-y: scroll;
+  margin: 0 auto;
+
+  ::-webkit-scrollbar {
+    width: 0;
+  }
+`;
+
+const Footer = styled.footer`
+  display: flex;
+  align-items: center;
+  justify-content: flex-end;
+`;
+
+const Copyright = styled.p`
+  margin: 0;
+  padding: 10px 7.5%;
```

Comment:
What is the reason for scaling the padding top / bottom differently from scaling the left / right?
If there is no strong reason please do not mix absolute and relative units like that.

Reply:
Lost only padding-rigth: 7.5%.